test/e2e_node/jenkins/copy-e2e-image.sh

```diff
@@ -16,10 +16,15 @@
 
 # Usage: copy-e2e-image.sh <image-name> <from-project-name> <to-project-name>
 
+# See *.properties for list of images to copy,
+# typically from kubernetes-node-e2e-images
+
 set -e
 set -x
 
 echo "Copying image $1 from project $2 to project $3..."
 gcloud compute --project $3 disks create $1 --image=https://www.googleapis.com/compute/v1/projects/$2/global/images/$1
-gcloud compute --project $3 images create $1 --source-disk=$1
-gcloud compute --project $3 disks delete $1
+gcloud compute --project $3 images create $1 \
+  --source-disk=$1 \
+  --description="Cloned from projects/$2/global/images/$1 by $USER on $(date)"
+gcloud -q compute --project $3 disks delete $1
```

Comment:
Skips asking for confirmation. We don't want the disk (created in line 26) but the easiest way to create an image is from a disk. So we use the temporary disk to create the image.